plugins/hako/readme.md

```diff
@@ -23,10 +23,25 @@ plugins:
 Deploy to ECS via the Hako CLI
 #### Task options
 
-| Property                | Description                                                       | Type            | Default |
-| :---------------------- | :---------------------------------------------------------------- | :-------------- | :------ |
-| `asReviewApp`           | whether to deploy as a temporary review app, used for code review | `boolean`       | `false` |
-| **`environments`** (\*) | the Hako environments to deploy an image to                       | `Array<string>` |         |
+| Property                  | Description                                                                                                                                                                 | Type            | Default |
+| :------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------------- | :------ |
+| `asReviewApp`             | whether to deploy as a temporary review app, used for code review. overrides the `customEphemeralId` option with its own hash of the git branch.                            | `boolean`       | `false` |
+| `customEphemeralId`       | ID that is used by Hako to identify a particular ephemeral app                                                                                                              | `string`        |         |
```

Comment:
question(non-blocking): should we just call this `ephemeralId`? and mention that the default is a hash when not provided? Also, keeps things consistent with say `HakoDelete` options _(except I'm misundertsanding)_

similarly, just `ephemeralManifest` as opposed to `customEphemeralManifest`?